### PR TITLE
Add 10 file system-based persistence techniques

### PR DESCRIPTION
**Pull request by jorik-utwente**
Add 10 file system-based persistence techniques.

All rules have a reference to an online resource describing the technique.

This PR requires https://github.com/mandiant/capa-rules/pull/952 to be merged.

### CODE REVIEW

File: nursery/persist-via-startup-folder.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  features:
+    - and:
+      - or:
+        - match: copy file
+        - match: move file
+        - match: host-interaction/file-system/write
+      - or:
+        - string: /Microsoft\\Windows\\Start Menu\\Programs\\Startup\\/i
+        - string: /Documents and Settings\\All Users\\Start Menu\\Programs\\Startup\\/i
+        - string: /WINNT\\Profiles\\All Users\\Start Menu\\Programs\\Startup\\/i
```
Comment (mr-tz) [inline nursery/persist-via-startup-folder.yml:23]: let's add these to `reference-startup-folder.yml` instead (and merge the rules)

**Review by mr-tz — COMMENTED** [review]
First off, sorry for the delay in getting to this. I've been busy but this is great work here and I'm excited for all these new capability detections! Please see my suggestions and comments for discussion inline.

File: nursery/persist-via-startup-folder.yml (hunk 1 above)
Comment (jorik-utwente) [reply nursery/persist-via-startup-folder.yml]: I'm removing this rule, since it is already part of https://github.com/mandiant/capa-rules/pull/953

**Review by mr-tz — APPROVED** [review]
Great, thanks a lot! I really appreciate all your updates, the discussions, and your proper git workflow!

**Comment by mr-tz** [comment]
LGTM!
Leaving this open for 1-2 more days to give Mike and Willi a chance to chime in if they want.

**Review by mike-hunhoff — APPROVED** [review]
Great work, LGTM 🚀

**Review by williballenthin — APPROVED** [review]

**Comment by williballenthin** [comment]
🎉

**Pull request merged**